staging/src/k8s.io/api/apps/v1beta1/types.go

```diff
@@ -58,6 +58,7 @@ type ScaleStatus struct {
 
 // +genclient=true
 // +noMethods=true
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
Yes, can be fixed via that tag. It's document in gengo https://github.com/kubernetes/gengo/blob/master/examples/deepcopy-gen/main.go#L17. Do we have any better place for this kind of docs?